vms/authentication/templates/registration/password_change_done.html

```diff
@@ -0,0 +1,13 @@
+{% extends "vms/base.html" %}
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">Home</a>
+Password change
+</div>
+{% endblock %}
+
+{% block title %}{{ title }}{% endblock %}
+{% block content_title %}<h1>{{ title }}</h1>{% endblock %}
+{% block content %}
+<p>Your password was changed.</p>
```

Comment:
VMS would be merged w/ Portal going forward, could you look for messages in Portal for changing password and update these. Also you need to load i18n file for enabling translation.

Reply:
@tapasweni-pathak The portal does not show up any message on password change. It just reloads the password change form. I think that the portal should come up with this feature.